drivers/memc/memc_mcux_flexspi.c

```diff
@@ -174,8 +181,16 @@ static int memc_flexspi_init(const struct device *dev)
 #define MEMC_FLEXSPI_CFG_XIP(node_id) false
 #endif
 
+#ifdef CONFIG_PINCTRL
+#define PINCTRL_DEFINE(n) PINCTRL_DT_INST_DEFINE(n);
```

Comment:
```suggestion
#define PINCTRL_DEFINE(n) PINCTRL_DT_INST_DEFINE(n)
```

Reply:
See below comment-if you feel strongly about this change I'm happy to implement it, but otherwise would prefer to keep this consistent with the other NXP driver implementations.